python/src/aac/plugins/validators/_validator_result.py

```diff
@@ -37,4 +37,13 @@ def format_message(finding: ValidatorFinding) -> str:
                 f"at {loc.location.line + 1}:{loc.location.column} {finding.message}"
             )
 
-        return "\n".join([format_message(finding) for finding in self.findings.get_all_findings()])
+        if len(self.findings.get_error_findings()) == 0:
+            definition_sources = []
+            for definition in self.definitions:
+                if definition.source.uri not in definition_sources:
+                    definition_sources.append(definition.source.uri)
+
+            return "\n".join(f"The definitions in {definition_file}" for definition_file in definition_sources)
+
+        else:
+            return "\n".join([format_message(finding) for finding in self.findings.get_all_findings()])
```

Comment:
This will cause us to lose information if the definitions are valid but, for example, `WARNING` or `INFO` findings were added to the result.

I can't guarantee this is exactly the code we want here but I think something like this would work (though haven't actually tested it to see). The goal is to show the validated files (assuming there were no `ERROR` messages) in addition to all the `WARNING` and `INFO` messages:

```suggestion
            valid_files_output = "\n".join(f"{definition_file} is valid" for definition_file in definition_sources)

        return valid_files_output + "\n".join([format_message(finding) for finding in self.findings.get_all_findings()])
```